README.rst

```diff
@@ -48,7 +48,7 @@ Parsing a TOML file is as easy as using the ``TOML.Load_File`` function:
 
    declare
       Result : constant TOML.Read_Result :=
-         TOML.Load_File ("config.toml");
+         TOML.Text_IO.Load_File ("config.toml");
```

Comment:
This one escaped renaming ;-)

Reply:
And I was so happy for having caught it the first time...

Reply:
Congrats for that, I wouldn’t have otherwise. ;-)